src/medication-dispense/medication-dispense.resource.tsx

```diff
@@ -39,16 +39,20 @@ export function initiateMedicationDispenseBody(
     let dispense = {
       resourceType: "MedicationDispense",
       status: "on-hold", // might need to change this to appropriate status
+      authorizingPrescription: {
+        reference: "MedicationRequest/" + medicationRequest.id,
+        type: "MedicationRequest",
+      },
       medicationReference: medicationRequest.medicationReference,
       medicationCodeableConcept: medicationRequest.medicationCodeableConcept,
       subject: medicationRequest.subject,
-      performer: [
+      /* performer: [
```

Comment:
I just commented this out for now, so we don't forget... the person who dispenses the medication should not be defaulted to the person who prescribed the medication... we can figure out the appropriate person to include here later.